Sources/GoogleAI/GenerateContentError.swift

```diff
@@ -24,4 +24,11 @@ public enum GenerateContentError: Error {
 
   /// A response didn't fully complete. See the `FinishReason` for more information.
   case responseStoppedEarly(reason: FinishReason, response: GenerateContentResponse)
+
+  /// The user's location (region) is not supported by the API.
+  ///
+  /// See the Google AI for Developers documentation for a
```

Comment:
I think we should remove "Google for AI Developers" in this sentence.

For example, in the README, we refer to the documentation as "the Google documentation":

https://github.com/google/generative-ai-swift/blob/5cf2aae0f4d04b77fe7b2110da1145462ff92b84/README.md?plain=1#L76

Reply:
Thanks for the feedback, @peterfriese! I reworded the documentation a bit, please let me know if this sounds alright. This is how it looks in Xcode:
![unsupportedUserLocation Error Screenshot](https://github.com/google/generative-ai-swift/assets/3010484/53a6b4ce-a706-42fb-8d02-57713b1f34e0)

I also like the idea of adding example usage in the sample app(s).

Note: I'm leaving this conversation as unresolved (to prevent accidentally merging) while we consider consistency across the SDKs internally. Will ping you the thread about it but this gets thrown as a FailedPrecondition exception the Python SDK (details below):
```
Exception Type: <class 'google.api_core.exceptions.FailedPrecondition'>
Exception Arguments: ('User location is not supported for the API use.',)
Exception Instance: 400 User location is not supported for the API use. [detail: "[ORIGINAL ERROR] generic::failed_precondition: User location is not supported for the API use. [google.rpc.error_details_ext] { message: \"User location is not supported for the API use.\" }"
]
```